### DISCUSSION TITLE
How to create demuxlet .best files for Mix of melanoma cell lines tutorial?

### DISCUSSION THREAD

**Discussion by MVemon** [aertslab/scenicplus#332] [Q&A] [answered]
Hello,

I am following the melanoma cell line pipeline. but encountered problems with producing the demuxlet.best files. I understand that they require bam and vcf files to produce .best files, but I am not sure how to produce those vcf files. After looking at freebayes and getk tutorials, creating vcf files also require bam files. It's all kinda confusing with which exact bam files need to be used.

Would love to hear any suggestions from people who have done this and what their steps were or if there is another way to link separate scRNA and scATAC data without using demuxlet.

Thanks.

**Comment by SeppeDeWinter** [comment, marked as answer]
Hi @marck1198 

Does you data contain a mix of different genotypes? It is only necessary to run demuxlet if this is the case.

I would recommend to have a look at the demuxlet repository: https://github.com/statgen/demuxlet.

Best,

Seppe

**Reply by MVemon** [reply]
Hello,

I realized that I didn't actually need to use demuxlet. I thought it was necessary to transfer annotation between the scRNA and scATAC, but I just used seurat to transfer annotations between data.

Sorry for the confusion and thanks for the help.

**Reply by SeppeDeWinter** [reply]
Hi Marck

No worries.

All the best,

Seppe
